ui/controllers/userController.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
```

Comment:
Apparently, `use strict` is no longer necessary. So, I've read anyways. I have to remove it from my own projects, too